app/src/main/res/values-in/strings.xml

```diff
@@ -0,0 +1,72 @@
+    <string name="write_a_review">Tulis ulasan</string>
+    <string name="source_code">Kode sumber</string>
+    <string name="features">Fitur - fitur</string>
+    <string name="support_this_project">Dukung proyek ini</string>
+    <string name="report_a_bug">Laporkan bug</string>
+    <string name="version_x">Versi %1$s</string>
+    <string name="app_description">%1$s adalah open source project yang dikembangkan oleh Lorenzo Vainigli dan kontributor lainnya. Kode aplikasi ini diterbitkan di bawah lisensi MIT dan tersedia di GitHub.</string>
+    <string name="about_this_app">Tentang aplikasi ini</string>
+    <string name="privacy_policy">Kebijakan privasi</string>
+    <string name="contributors_list_title">Kontributor</string>
+    <string name="contributors_list_subtitle">Orang - orang yang berkontribusi terhadap pengembangan aplikasi ini.</string>
+    <string name="back">Kembali</string>
+    <string name="no_items_found">Tidak ada item yang ditemukan</string>
+    <string name="please_insert_one">Silakan masukkan satu item</string>
+    <string name="menu">Menu</string>
+    <string name="settings">Pengaturan</string>
+    <string name="date_format">Format tanggal</string>
+    <string name="choose_the_date_format">Pilih format tanggal</string>
+    <string name="locale_formats">Format lokal</string>
+    <string name="other_formats">Format lainnya</string>
+    <string name="your_food_is_expiring">Makanan anda kadaluarsa!</string>
+    <string name="notification_time">Waktu notifikasi</string>
+    <string name="dynamic_colors">Warna dinamis</string>
+    <string name="theme">Tema</string>
+    <string name="light">Terang</string>
+    <string name="system">Sistem</string>
+    <string name="dark">Gelap</string>
+    <string name="x_deleted">%1$s dihapus</string>
+    <string name="undo">Batalkan</string>
+    <string name="normal">Normal</string>
+    <string name="bold">Tebal</string>
+    <string name="extra_bold">Ekstra Tebal</string>
+    <string name="top_bar_font_style">Gaya font bilah atas</string>
+    <string name="behaviour">Perilaku</string>
+    <string name="appearance">Tampilan</string>
+    <string name="send_an_email">Kirim email</string>
+    <string name="contacts_text">Jika anda ingin mendapatkan informasi lebih lanjut tentang aplikasi ini, kirim umpan balik, laporkan bug, request fitur, atau hubungi pengembang, silakan kirim email menggunakan tombol berikut.</string>
+
+    <string-array name="features">
+        <item>Display a list with food expiration dates in ascending time order.</item>
+        <item>Add, edit and delete items.</item>
+        <item>Notifications.</item>
+    </string-array>
+    <string-array name="example_foods">
+        <item>Eggs</item>
+        <item>Cheese</item>
+        <item>Milk</item>
+        <item>Ham</item>
+        <item>Butter</item>
+        <item>Mushrooms</item>
+        <item>Tomatoes</item>
+    </string-array>
```

Comment:
Could you translate also these strings?

Reply:
Oh, I forgot the bottom side cause I used Translations Editor in Android Studio.
I'm sorry i'ts my first time contribute like localization and edit them directly.

Next commit will be great fix.

Reply:
There's no problem at all, don't worry! I'm happy to help contributors 😉